modules/local/sentieon/bqsr.nf

```diff
@@ -9,8 +9,8 @@ process SENTIEON_BQSR {
     tuple val(meta), path(bam), path(bai)
     path fasta
     path fai
-    path known_dbsnp
-    path known_dbsnp_tbi
+    tuple val(meta2), path(known_dbsnp)
```

Comment:
What are meta2 ans meta3? I mean why do we need them?

Reply:
They don't do anything here in this case. But dbsnp file is expected with metadata elsewhere and I thought its easier to have it be consistent across all modules. Otherwise we would need create another channel for dbsnp file without metadata. 😄